palo_alto_panorama/datadog_checks/palo_alto_panorama/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.0.1'
```

Reply:
Why version 0.0.1?

Reply:
The bump to 1.0.0 will be done as part of the Agent release process, when we will release the integration

Reply:
Ok.

Reply:
Done, Updated to `0.0.1` version.